lib/Listerine/commands.ex

```diff
@@ -80,4 +83,17 @@ defmodule Listerine.Commands do
 
     Message.reply(message, text)
   end
+
+  command help() do
+    embed = %{
+      title: "Commands",
```

Comment:
The commands descriptions are in Portuguese but the title is in English? Shouldn't we keep it all in the same language?